test/extended/operators/service_catalog.go

```diff
@@ -0,0 +1,161 @@
+package operators
+
+import (
+	"fmt"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+var _ = g.Describe("[Feature:Platform] Service Catalog should", func() {
+	defer g.GinkgoRecover()
+
+	var oc = exutil.NewCLI("service-catalog", exutil.KubeConfigPath()).WithoutNamespace()
```

Comment:
it's possible this is your issue.  Why are you choosing "withoutnamespace"?  This means your oc invocations will not have a `-n` arg added to them for the namespace your e2e test is running in.

Reply:
I thought I have to add `--namespace` value for each `oc` command if I don't use `WithoutNamespace()`. So, I used it.

Reply:
i think you have that backwards.

Reply:
Remove the `WithoutNamespace()` func and it works now, thanks!